### DISCUSSION TITLE
Using PnP Search with document sets in Sahrepoint

### DISCUSSION THREAD

**Discussion by AdoptionDigitale** [microsoft-search/pnp-modern-search#3502] [Q&A] [answered]
Hi team, 

I need some help... I have a sharepoint document library with document sets enabled.
My customer requests to search for those documents.
I have set the Search Box webpart, a Serach Results webpart and a Filters webpart.
The Search results display all the document sets as per my query... and that's fine.
The filters work fine as well with the serach results webpart.
But I am unable to make the serach results work with the search box.
Any time, I am entering a keyword, il returns 0 results.

Any idea where I did wrong ?
thank you for your help

**Comment by kasperbolarsen** [comment]
I am sorry, but we can't help you if you don't help us.  Please follow the issue template and in your case, we also need to know the query template and perhaps a screen shot of page 3 in the result web part.

**Comment by AdoptionDigitale** [comment]
Oupps, some additional information

- SharePoint online site : Team site template
- Doc lib (documentsRTCS) with Documents set name : Dossier de Référence
- lookup columns in doc lib based on 3 lists with lookUp columns : Parties(Parent), Catégories (child of Parties), Sous-catégories (child of Catégories)
- Managed properties mapped with crawled properties (Parties, Catégories, Sous-catégories)

<img width="260" alt="PnPSearchResults-Page3" src="https://github.com/microsoft-search/pnp-modern-search/assets/74817536/a35f1a08-75b1-4364-ba10-c7202799ca51">
<img width="236" alt="PnPSearchResults-QueryTemplate" src="https://github.com/microsoft-search/pnp-modern-search/assets/74817536/5341bd25-e8ae-4f92-849e-6c53ea81b294">
<img width="246" alt="PnPSearchResults-QueryFilter" src="https://github.com/microsoft-search/pnp-modern-search/assets/74817536/e8b1a7ff-8958-4179-947a-94a2cb1c955d">
[Results Template.txt](https://github.com/microsoft-search/pnp-modern-search/files/14151689/Results.Template.txt)

**Comment by gretchunkim** [comment]
hmm. If I  recall, I believe document sets belongs to document tag/metadata category  of the search. You might want to  search based on that keyword so you can see if the answer is already posted.

**Comment by IngoHenning** [comment, marked as answer]
In your query template try to add a space after {searchTems} and it should work. With your template a search for the word Test would result in in TestPath:https://... that cannot work

**Reply by wobba** [reply]
..and you can also add the ContentType FQL as KQL in the query template if you want.. no need to use both.

**Reply by AdoptionDigitale** [reply]
Thank you so much. It works now!
